.github/ISSUE_TEMPLATE/New_Currency_Request.yml

```diff
@@ -0,0 +1,45 @@
+name: New Currency Request
+title: "[Currency_Req]: "
+description: "Request to add currency for any new package with specific version on ppc64le"
```

Comment:
Here we are saying "ppc64le" instead of "power", lets decide and be consistent

Reply:
I vote for Linux on Power - we have enough questions about AIX and IBM i already, and not everyone understands the ppc64le term completely.